src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -145,10 +145,6 @@ export class ItemListLayout<I extends ItemObject> extends React.Component<ItemLi
       throw new Error("[ItemListLayout]: configurable list require props.tableId to be specified");
     }
 
-    if (isConfigurable && !UserStore.getInstance().hiddenTableColumns.has(tableId)) {
-      UserStore.getInstance().hiddenTableColumns.set(tableId, new ObservableToggleSet());
-    }
-
```

Comment:
this is getting done now elsewhere?

Reply:
Yes, it is being done https://github.com/lensapp/lens/pull/3310/files/97776b9c48b8e549ea5390c3bd4cd1833a7d43a2#diff-dcf7a988ad98b11a4537826fdb986a5ef56f35bdc827e66285c57138446caacaR174-R177